skipper.go

```diff
@@ -1167,9 +1173,13 @@ func (o *Options) tlsConfig(cr *certregistry.CertRegistry) (*tls.Config, error)
 	return config, nil
 }
 
-func (o *Options) openTracingTracerInstance() (ot.Tracer, error) {
+func (o *Options) tracerInstance() (trace.Tracer, error) {
```

Comment:
I don't see that we use this function in skipper.go. As far as I understand we should use this function to set "tracer" in run() or not?

Reply:
If we only use this func in test we can likely move it to a _test.go file

Reply:
Hmm yeah, this should be used in the `run()` function, it actually seems to me that we are missing the definition of tracer in the options. in the `run()` function and even if it was correct there we would be just duplicating code, since all these checks are done in `tracerInstance(...)`anyway.